presto-hive/src/main/java/com/facebook/presto/hive/BackgroundHiveSplitLoader.java

```diff
@@ -421,6 +430,58 @@ private void loadPartition(HivePartitionMetadata partition)
         fileIterators.addLast(iterator);
     }
 
+    private void addLzoThriftSplitsToQueue(
+            HiveFileIterator hiveFileIterator,
+            String partitionName,
+            Properties schema,
+            List<HivePartitionKey> partitionKeys,
+            ConnectorSession session,
+            TupleDomain<HiveColumnHandle> effectivePredicate,
+            Map<Integer, HiveType> columnCoercions)
+            throws IOException
+    {
+        if (bucketHandle.isPresent()) {
+            throw new PrestoException(StandardErrorCode.NOT_SUPPORTED, "Bucketed table in ThriftGeneralInputFormat is not yet supported");
```

Comment:
I see. It's basically a field-partitioned table. We should include the table name in the exception message.